marketplace/shared-dam-app/src/Editor/SortableComponent.tsx

```diff
@@ -83,21 +83,47 @@ const SortableItem = SortableElement<SortableElementProps>((props: SortableEleme
 });
 
 const SortableList = SortableContainer<SortableContainerProps>((props: SortableContainerProps) => {
+  // Provide stable keys for all resources so images don't blink.
+  const { list } = props.resources.reduce(
+    (acc, resource, index) => {
+      const [url, alt] = props.makeThumbnail(resource, props.config);
+      const item = { url, alt, key: `url-unknown-${index}` };
+      const counts = { ...acc.counts };
+
+      // URLs are used as keys.
+      // It is possible to include the same image more than once.
+      // We count usages of the same URL and use the count in keys.
+      // This can be considered an edge-case but still - should be covered.
+      if (url) {
+        counts[url] = counts[url] || 1;
```

Comment:
Should the default be `0` here instead of `1`? You are incrementing on line 100.

Reply:
doesn't really matter, I guess my keys are 1-indexed ;)